src/util/alloc/immix_allocator.rs

```diff
@@ -304,12 +306,14 @@ impl<VM: VMBinding> ImmixAllocator<VM> {
                     block.start(),
                     block.end()
                 );
+                let (cursor, limit) =
+                    adjust_thread_local_buffer_range::<VM>(block.start(), block.end());
```

Comment:
Is there any reason to adjust block start/end addresses? Looks like Java Immix does not need this?

https://github.com/JikesRVM/JikesRVM/blob/master/MMTk/src/org/mmtk/utility/alloc/ImmixAllocator.java#L192-L198

Reply:
Is there a fix that I can do for this in this PR?

Reply:
This code is removed in https://github.com/mmtk/mmtk-core/pull/699.